Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/XmlDocumentationCodeFixProvider.cs

```diff
@@ -14,7 +14,7 @@
 namespace Philips.CodeAnalysis.MaintainabilityAnalyzers.Documentation
 {
 	//	[ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(EmptyXmlCommentsCodeFixProvider)), Shared]
```

Comment:
This is a great candidate for auto-correct, because there are so many findings in a large code base, and it's relatively safe to auto-change it without creating a code logic change. But my memory is that I couldn't get the Code Fixer to work properly for this analyzer, though to be fair this may have been my first attempt at a code fixer x years ago.